### ISSUE TITLE
Colyseus multiplayer engine alternative

### ISSUE THREAD

**Issue by DreadKnight** [alto-io/game3.js#3]
Will Nakama also be supported eventually?
https://github.com/heroiclabs/nakama

**Comment by polats** [comment]
yes! I actually started with using nakama on an earlier version (https://github.com/alto-io/funding-the-future), but decided to prioritize colyseus since it's mostly javascript. That might be a good project for the [unitize hackathon](https://gitcoin.co/hackathon/unitize) actually, I'll create a bounty on it

**Comment by DreadKnight** [comment]
Awesome, good to hear! Yeah, it's cool that Colyseus is JS only, as it can go about faster implementing integrations with 3rd party stuff. I'm all about this game3.js kind of stuff in the long run for my projects, especially my main one https://github.com/FreezingMoon/AncientBeast

As a side note, I really hope one of the foss/js engines will implement decentralized accounts, so far Nakama rejected due to lack of interest https://github.com/heroiclabs/nakama/issues/436 but I've found out about web3modal, which might make more sense https://github.com/Web3Modal/web3modal (I've just noticed that it supports 3box accounts as well, nice!).

**Comment by polats** [comment]
thinking of using 3box also! I've seen some of the web3 projects starting to integrate it, but am still weighing out the need for it vs. just having web3modal

**Comment by DreadKnight** [comment]
Cool, I think it's good stuff, a really needed alternative to fb login and such. But if I were a good coder, would definitely rather implement web3modal instead, as it adds the needed abstraction layer for this type of things instead of just fixating on one.

**Comment by polats** [comment]
Good to know, thanks a lot! For now I'll build on the web3modal library first and maybe it can become easy enough for everyone to use

**Comment by DreadKnight** [comment]
Cheers! Keep up the good work, we really need to push the industry in this decentralized / open source direction and free ourselves from the tyranny of the capitalistic corporate machine that feeds on us :D
